safe_transaction_service/history/services/index_service.py

```diff
@@ -323,7 +323,7 @@ def _reprocess(self, addresses: List[str]):
 
         logger.info("Remove transactions automatically indexed")
         queryset = MultisigTransaction.objects.exclude(ethereum_tx=None).filter(
-            Q(origin=None) | Q(origin="")
+            Q(origin__exact={}) | Q(origin="")
```

Comment:
We shouldn't have empty strings on database
```suggestion
            Q(origin__exact={})
```

Reply:
> We shouldn't have empty strings on database

@Uxio0  I agree but could be because "" is a valid json and the object creation accepts it, removing the query filter the following test is going to fail [L158](https://github.com/safe-global/safe-transaction-service/pull/1177/files#diff-ed6ce143344e1aefcda7bdce62a260fd78d4569e90ffbc4d097eea27f911b337L158) because when create the multisigtransaction the origin is "". Null origins is formatted to {} but empty strings "" doesn't change when insert.

Reply:
Then we might remove the test, as there's no way from user input to get a "" on database, or at least we are making our best to prevent that